README.md

```diff
@@ -1,70 +1,35 @@
-# Getting Started with Create React App
+# Weather App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+A weather app that allows the user to search for the weather in cities around the UK. There are small summaries of the day and the following four days showing the date, an icon of the weather, maximum temperature, and a small description. Beneath these summaries is a more detailed summary of the day with maximum temperature, minimum temperature, humidity, wind speed, and wind direction. The user can click on more details to see the detailed summary of each day listed.
 
-## Available Scripts
+## Motivation
 
-In the project directory, you can run:
+This app was built as part of my coursework at Manchester Codes. I practiced using React, useState, and useEffect.
 
-### `npm start`
+## Teach/framework used
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+### Built with:
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- React (Front-end framework)
+- Jest (Testing library)
+- Axios (Making HTTP requests to the API)
+- Prettier (Keeps code to the style guide)
```

Comment:
`prettier` is a dev dep, you could also add `proptypes` and `react-icons-weather` to this dependencies list